typescript/simple-mercari-web/src/components/ItemList.tsx

```diff
@@ -29,12 +30,16 @@ export const ItemList = ({ reload, onLoadCompleted }: Prop) => {
   }, [reload, onLoadCompleted]);
 
   return (
-    <div>
+    <div className = "ItemListContainer">
       {items?.map((item) => {
+        const imageUrl = item.image_name
+          ? `${SERVER_URL}/images/${item.image_name}`
+          : PLACEHOLDER_IMAGE;
         return (
           <div key={item.id} className="ItemList">
-            {/* TODO: Task 2: Show item images */}
-            <img src={PLACEHOLDER_IMAGE} />
+            <img src={imageUrl} 
+                style={{ width: '200px', height: '200px'}}
```

Comment:
[nits]
CSSを割り当てる場合は、Styleを直接書くのではなく、ClassNameを設定してCSS側でスタイルを一括設定するのが一般出来です！このようにStyleを直接書くと、設定を変えたい時に一括管理されていないので編集するときに手間がかかってしまいます 😢 

```suggestion
            <img src={imageUrl}  className="ItemImage" />
```

```css
/* index.css */
.ItemImage {
  width: '200px';
  height: '200px';
}